tests/test_checks.py

```diff
@@ -0,0 +1,142 @@
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+from django.core import checks
+
+from corsheaders.conf import corsheaders_settings
+from corsheaders.checks import check_settings
+
+from corsheaders.defaults import default_headers, default_methods
+
+class SettingsTests(SimpleTestCase):
+
```

Comment:
```python
with pytest.raises(SystemCheckError):
    call_command(...
```

(You could also use `self.assertRaises`, but since we're using `pytest`, it's nicer to use `pytest.raises` which has a better failure message).

Also why did you add the `ROOT_URLCONF` setting?